src/Microsoft.Diagnostics.Monitoring.WebApi/Controllers/ControllerExtensions.cs

```diff
@@ -17,6 +17,15 @@ namespace Microsoft.Diagnostics.Monitoring.WebApi.Controllers
 {
     internal static class ControllerExtensions
     {
+        public static ActionResult FeatureNotEnabled(this ControllerBase controller, string featureName)
+        {
+            return new BadRequestObjectResult(new ProblemDetails()
+            {
+                Detail = string.Format(Strings.Message_FeatureNotEnabled, featureName),
```

Comment:
Do these go into the log?

Reply:
The fact that the `/stacks` route returns a 400 is in the .NET Monitor log:

```bash
11:07:04 info: Microsoft.AspNetCore.Mvc.Infrastructure.ObjectResultExecutor[1]
      => SpanId:04669b05859332ce, TraceId:bb61993385c42233241779b9fb41bc96, ParentId:0000000000000000 => ConnectionId:0HMUB9O535RFL => RequestPath:/stacks RequestId:0HMUB9O535RFL:00000001 => Microsoft.Diagnostics.Monitoring.WebApi.Controllers.DiagController.CaptureStacks (Microsoft.Diagnostics.Monitoring.WebApi)
      Executing BadRequestObjectResult, writing value of type 'Microsoft.AspNetCore.Mvc.ProblemDetails'.
11:07:04 info: Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker[105]
      => SpanId:04669b05859332ce, TraceId:bb61993385c42233241779b9fb41bc96, ParentId:0000000000000000 => ConnectionId:0HMUB9O535RFL => RequestPath:/stacks RequestId:0HMUB9O535RFL:00000001
      Executed action Microsoft.Diagnostics.Monitoring.WebApi.Controllers.DiagController.CaptureStacks (Microsoft.Diagnostics.Monitoring.WebApi) in 36.7633ms
11:07:04 info: Microsoft.AspNetCore.Routing.EndpointMiddleware[1]
      => SpanId:04669b05859332ce, TraceId:bb61993385c42233241779b9fb41bc96, ParentId:0000000000000000 => ConnectionId:0HMUB9O535RFL => RequestPath:/stacks RequestId:0HMUB9O535RFL:00000001
      Executed endpoint 'Microsoft.Diagnostics.Monitoring.WebApi.Controllers.DiagController.CaptureStacks (Microsoft.Diagnostics.Monitoring.WebApi)'
11:07:04 info: Microsoft.AspNetCore.Hosting.Diagnostics[2]
      => SpanId:04669b05859332ce, TraceId:bb61993385c42233241779b9fb41bc96, ParentId:0000000000000000 => ConnectionId:0HMUB9O535RFL => RequestPath:/stacks RequestId:0HMUB9O535RFL:00000001
      Request finished HTTP/1.1 GET https://localhost:5001/stacks - 400 - application/problem+json;+charset=utf-8 115.6026ms
```